sklearn/feature_extraction/hashing.py

```diff
@@ -123,6 +125,15 @@ def fit(self, X=None, y=None):
         """
         # repeat input validation for grid search (which calls set_params)
         self._validate_params(self.n_features, self.input_type)
+
+        # optional if random_state is left to an integer seed
+        if isinstance(self.random_state, int):
+            self.seed_ = self.random_state
```

Comment:
I feel like this branch distinction is unnecessary.  
If random_state is an int, then doing the steps below will put this int in `self.seed_`